libs/types/src/tokens.rs

```diff
@@ -255,6 +255,25 @@ impl CrossChainTransferability {
 	}
 }
 
+/// Connectors-wrapped tokens
+///
+/// Currently, Connectors are only deployed on EVM-based chains and therefore
+/// we only support EVM tokens. In the far future, we might support wrapped
+/// tokens from other chains, such as Cosmos, Avalanche, etc.
```

Comment:
```suggestion
/// tokens from other chains, such as Cosmos based ones.
```

Avalanche is EVM based.

Reply:
Thanks 👍